Automattic/jetpack
# Blocks: move localizable strings out of the metadata attributes field

Fixes https://github.com/Automattic/jetpack/issues/36649

## Proposed changes:

Block attributes defined in the metadata `block.json` file are not localizable (see [the WordPress.org documentation](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#attributes)). This PR updates the following blocks to remove translatable strings from this file:
- Paid Content
- Jetpack AI Search
- Donations
- Subscriptions
- Cookie Consent
- Goodreads
- Mailchimp

The approach is to remove default strings from the `block.json` files, define them in a JS file instead, and use them as a fallback when reading the attributes.

### Other information:


## Jetpack product discussion


Project thread: pf5801-Aw-p2

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






For each block above, verify that it still works as expected in the editor and in the site (the Cookie Consent block may only be added in the site editor). Ideally, change the language to a language other than English and check that all UI elements are displayed in that language.

_Note: if you can't see the text in the language you chose, make sure to update the latest translations in the Update page. You can also check if the translation exists with [this tool](https://translate.wordpress.org/projects/wp-plugins/jetpack/stable/)._
